blocks-desktop/b-page/b-page.examples/70-csp-with-nonce.bemjson.js

```diff
@@ -0,0 +1,21 @@
+([
+    {
+        block: 'b-page',
+        title: 'Page with CSP meta-tag',
+        nonce: 'test-nonce-value',
+        csp: {
+            policies: {
+                'script-src': [ 'examples.com' ],
+                'style-src': ["'self'", "'unsafe-inline'"]
```

Comment:
Maybe templates will be too complicated, because new quoted values may be added by w3c in the future.